### ISSUE TITLE
Share updated graphics with MoCo

### ISSUE THREAD

**Issue by edrushka** [mozilla/donate.mozilla.org#531]
Let's loop MoCo into where we're at with design.

@lovegushwa, when you're done with board stuff, can you put together a couple images showing the new graphic work (friendly nudge text, updated share and email signup pages on the form). I don't expect any issues, but just want to make sure they're kept in the loop, as we promised.

cc @valianttry

**Comment by edrushka** [comment]
@valianttry the Moco team (Jean, Jessica, MJ, Michaela and Ben) were invited by email on Sept 28 to take a look at [this Redpen] (https://redpen.io/foca6e5677782f60b1) and the [funnel designs] (https://drive.google.com/a/mozilla.com/file/d/0B2qu3ExKWIlvR3Q4WWdyRmZuQms/view?usp=sharing).

Do we need further looping in at this time?

**Comment by valianttry** [comment]
Hi @edrushka, No -- as long as their suggestions were addressed -- they seemed really minor and looks like Jordan responded to all of them. (We didn't ask for input on the funnel, just the snippet -- they're not really going to be engaged in our form design other than possibly the thank you/impact content TBD.)

One question: Have we landed on a treatment for the [x] opt out? That was one thing that seemed a bit up in the air... but I may just be out of the loop.
